tests/containers/helm.pm

```diff
@@ -14,14 +14,37 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use mmapi 'get_current_job_id';
 
+
+sub install_kubectl {
+    my $version = get_var('PUBLICCLOUD_KUBECTL_VERSION', 'v1.22.2');
+    assert_script_run("curl -LO https://dl.k8s.io/release/$version/bin/linux/amd64/kubectl");
+    assert_script_run("chmod a+x kubectl");
+    script_run("rm /usr/bin/kubectl");
+    assert_script_run("mv kubectl /usr/bin/kubectl");
```

Comment:
Feels like these 3 lines can be combined into one `install -c -m 644 ./kubectl /usr/bin/kubectl`

Reply:
Done